### ISSUE TITLE
Can Node 20 be added to the list of approved engines?

### ISSUE THREAD

**Issue by eldang** [jonataswalker/ol-contextmenu#268]
When upgrading from 4.x to 5.x, I got the following error:

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'ol-contextmenu@5.2.1',
npm WARN EBADENGINE   required: { node: '^16 || ^18', npm: '>=8' },
npm WARN EBADENGINE   current: { node: 'v20.5.0', npm: '9.8.0' }
npm WARN EBADENGINE }
```

When I decided to ignore that and proceed, the site I built seems to be working just fine.  Is there any chance we could either add node 20 to the list of supported engines, or switch it to `'>=16'`?

I am willing to make a PR to this effect, but I have no experience writing Typescript, and don't have a clear sense of what additional testing you'd want done.  If you can point me to any guidance about the latter, and specify if you have a preference between adding 20 or making it accept all `>=16`, I'm happy to try.

**Comment by jonataswalker** [comment]
Hi @eldang,

Yes, a PR to fix this will be great. There's no much to change. I guess `package.json` and [`test.yml`](https://github.com/jonataswalker/ol-contextmenu/blob/master/.github/workflows/test.yml#L19) is enough.

And yup, `>=16` is better in `package.json#engines#node`.

**Comment by eldang** [comment]
Cool, I should be able to do that next week.

**Comment by jonataswalker** [comment]
Closed by #269
